js/historyview.js

```diff
@@ -951,6 +958,15 @@ define(['d3'], function () {
                 throw new Error('Already up-to-date.');
             } else if (currentCommit.parent2 === mergeTarget.id) {
                 throw new Error('Already up-to-date.');
+			} else if (noFF === true) {
```

Comment:
I must have done this with a different editor, normally always have spaces :/

Sent from my Windows Phone
________________________________
From: Wei Wang<mailto:notifications@github.com>
Sent: ý25/ý03/ý2015 14:13
To: onlywei/explain-git-with-d3<mailto:explain-git-with-d3@noreply.github.com>
Cc: Jake Ginnivan<mailto:jake@ginnivan.net>
Subject: Re: [explain-git-with-d3] No fast forward merge (#32)


In js/historyview.js<https://github.com/onlywei/explain-git-with-d3/pull/32#discussion_r27123460>:

> @@ -951,6 +958,15 @@ define(['d3'], function () {
>                  throw new Error('Already up-to-date.');
>              } else if (currentCommit.parent2 === mergeTarget.id) {
>                  throw new Error('Already up-to-date.');
> +                     } else if (noFF === true) {


still have mixed tabs and spaces here. I really need to install a linter.

—
Reply to this email directly or view it on GitHub<https://github.com/onlywei/explain-git-with-d3/pull/32/files#r27123460>.